text/0000-issue-workflow.md

```diff
@@ -0,0 +1,222 @@
+     |                         |                    E  |                         |
+     |                         |                    S  |                         |
+     |             +-----------v----------+         T  |             +-----------v----------+
+     |             |                      |         S  |             |                      |
+     |             |  status/in-progress  |            |             | status/user-accepted |
+     |             |                      |            |             |                      |
+     +----------+  +----------------------+            +----------+  +----------------------+
+                               |                                                 |
+                               |                                                 |
+                               |                                                 |
+                               v                                                 v
+                             CLOSED                                            CLOSED
+```
+
+
+### Actionable Issues
+
+An actionable issue is one that can be worked on such that the following is true:
+
+- Definition of done - that may be acceptance criteria or clear expectations.
+- Appropriate labels
+    - `type/*`
+    - `size/*` (if possible)
+    - `status/ready`
+    - `good first issue` (if applicable)
+
+
+### Triage
+
+The following is a flow assist contributors in assigning the appropriate labels in script format.
+
+#### Issues
+
+- Is this a support request?
+    - Yes
+        - Add label type/support
+        - Provide support and close the issue with label `status/resolved`\
+          OR 
+        - Request further information from the issue author and label `status/incomplete`
+        - If activity is missing for 7 business days, follow up or close
+    - No
+        - See below
+- Is this something we want to do?
+    - I don’t even understand this issue
+        - Remove label `status/triage`
+        - Add label `status/invalid`
+        - Comment and close
+    - No - not valid / relevant for our project
+        - Remove label status/triage
+        - Add label `status/not-planned` or `status/invalid`
+        - Comment and close
+    - Maybe - needs RFC
+        - Remove label `status/triage`
+        - Add label `status/requires-rfc`
+        - Comment and close
+    - Maybe - needs alignment
+        - Remove label `status/triage`
+        - Add label `status/discussion-needed`
+        - Add appropriate `type/*`
+    - Maybe - I can’t tell from the information in the issue
+        - Remove label `status/triage`
+        - Add label `status/incomplete`
+        - Request further information from issue author
+        - If activity is missing for 7 business days, follow up or close
+    - Yes - but it’s not on our roadmap (would consider a PR)
+        - Remove label `status/triage`
+        - Add label `status/not-planned`
+        - Comment and close
```

Comment:
Why not leave this open?

Reply:
This might be dependent on opinion but typically open issues should convey items that have a path forward. If it's not planned as something we would actively strive for I personally don't see the reason for maintaining an issue open.  Typically these sort of issues tend to stay open indefinitely and just recently we have decided to close as inactive or stale.

Also, roadmap might be the wrong word here given that's it's not necessarily speaking about [ROADMAP.md](https://github.com/buildpacks/community/blob/master/ROADMAP.md) but rather a general view of where the project is going.

For some examples, here are some that have been previously closed with this scenerio in mind:
https://github.com/buildpacks/pack/issues?q=is%3Aissue+label%3Astatus%2Fnot-planned+is%3Aclosed

Reply:
Maybe it's the wording in the workflow that is throwing me here.

Some of the `not-planned` examples you linked seem more like a "No" or "Maybe" when answering the question "Is this something we want to do?", and this is the "Yes" case.

The workflow implies that this would apply to issues the maintainers believes to be a good idea and would accept a PR for - not necessarily issues that are misaligned with the direction of the project.

If we like the feature proposed in an issue, I feel like we should leave it open for some time and potentially use a bot to clean up inactive issues after a given window. That would indicate that we are supportive of the author or another contributor working on the issue even though it isn't a priority for the team.